lib/wallet/node.js

```diff
@@ -108,6 +108,12 @@ class WalletNode extends Node {
     await this.http.open();
     await this.handleOpen();
 
+    const clientInfo = await this.client.getInfo();
```

Comment:
`await` unneeded

Reply:
Can't remove it here, if wallet is running as a standalone service `getInfo()` is an actual HTTP API call. If you remove it here, the test will fail.